### PR TITLE
Installer prometh

### PR DESCRIPTION
**Pull request by jayunit100**

### CODE REVIEW

File: contrib/install/openshift/perceptor.yaml [hunk 1]
```diff
@@ -28,12 +28,13 @@ items:
               - ./perceptor
             ports:
               - containerPort: 3001
+                protocol: TCP
             resources:
               requests:
                 memory: 2Gi
-                cpu: 50m
+                cpu: 100m
```
Comment (mattfenwick) [inline contrib/install/openshift/perceptor.yaml]: Just curious, was perceptor not being allocated enough processor?  => would be great if this fixed the "connection refused" errors ...

File: contrib/install/install-os-kube-universal.sh [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+install-contrib() {
+	set -e
+	#### 
+	#### The perceptor core functionality 
+	####
+
+	pushd kube/
+		$KUBECTL create -f kube-perceiver.yaml --namespace=$NS
```
Comment (mattfenwick) [inline contrib/install/install-os-kube-universal.sh]: So we install the openshift-perceiver and the kube-perceiver when we're running on openshift?

Comment (rrati) [reply contrib/install/install-os-kube-universal.sh]: You shouldn't.  When running on openshift you should just install the openshift-perceiver, which contains the pod and image perceivers.

Comment (jayunit100) [reply contrib/install/install-os-kube-universal.sh]: i changed it !

**Pull request merged**